network/network.go

```diff
@@ -611,70 +611,86 @@ func (n *Network) PeerDiagnostics() map[transport.PeerID]transport.Diagnostics {
 }
 
 func (n *Network) Reprocess(contentType string) {
-	batchSize := uint32(1000)
+	go func() {
+		_, err := n.ReprocessContentType(context.Background(), contentType)
+		if err != nil {
+			log.Logger().Error(err)
+			return
+		}
+
+		// message in use by issue #1445
+		log.Logger().Infof("reprocess %q complete", contentType)
+	}()
+}
+
+// ReprocessReport describes the reprocess exection.
+type ReprocessReport struct {
+	// reserved for future use
+}
 
+func (n *Network) ReprocessContentType(ctx context.Context, contentType string) (*ReprocessReport, error) {
```

Comment:
shouldn't be exported?

Reply:
This should be used instead of the current Reprocess. Ideally just drop and replace, if that is something we can do?

Reply:
It's called from an API which is returning `202 accepted`. The API can't wait for the process since it'll timeout on the client side.

Reply:
Sure. The HTTP 202 coupling makes this launch, so that's where the `go` fork should be—not somewhere hidden in a subpackage.